Giveaways.md

```diff
@@ -0,0 +1,15 @@
+# Giveaways
+From time to time DevStaff has the opportunity to offer some giveaways during the meetups. The prizes are provided by [sponsors](https://github.com/devstaff-crete/DevStaff-Heraklion/blob/master/Sponsors.md) in return of advertising.
+
+The process, the tools and the rules we use for these giveaways are described here:.
+
+# Prizes list
+All prizes will be written down at each meetup's announcement at [meetup.com](https://www.meetup.com/devstaff/). Moreover, at the intro presentation, when sponsors are presented, each prize will be explicitly mentioned as far as the type, the amount of prizes and the sponsor of each one.
```

Comment:
Can you please replace `as far as` with  `as well as`?

Reply:
Fixed by https://github.com/devstaff-crete/DevStaff-Heraklion/pull/462/commits/f80fd6905054efa92aa01f37666a9f432a4ccff1